website/views.py

```diff
@@ -1411,6 +1418,11 @@ def get_context_data(self, *args, **kwargs):
             .annotate(c=Count("label"))
             .order_by()
         )
+        domain_name = Domain.get_name
+        context["twitter_url"] = "https://twitter.com/%s" + (
+            domain.get_twitter_account(domain_name)
```

Comment:
This code doesn't look right.

Reply:
please check new changes made. do they resolve this issue